modular_zubbers/master_files/skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -0,0 +1,127 @@
+// Precompiled first aid kits, ready to go if you don't want to bother getting individual items
```

Comment:
Ditto

Reply:
I went through the medical packs and calculated the value of items in each one. The prices where it was, it was giving out items far cheaper than medbay's own Nanomed. Medical getting a discounted version locked to themselves, and a more expensive one that's available to the general public seems fair.

Reply:
For example, being able to buy a laser scalpel, mechanical pinchers, and searing tool using your roundstart cash, an hour before the research will be finished for medbay, let alone materials, is insane.

Reply:
> For example, being able to buy a laser scalpel, mechanical pinchers, and searing tool using your roundstart cash, an hour before the research will be finished for medbay, let alone materials, is insane.
> 
> The cargo imports medical rebalance would have been my own PR, but Zender had this one so I added.

If you have a better plan for medical, I can atomize my changes to the medical items and you can take care of that, you play more medical than I do